tests/h/presenters/annotation_searchindex_test.py

```diff
@@ -123,7 +123,9 @@ def test_it_does_not_mark_annotation_hidden_when_not_moderated_and_no_replies(se
         assert annotation_dict['hidden'] is False
 
     def test_it_marks_annotation_hidden_when_moderated_and_no_replies(self, pyramid_request, moderation_service):
-        annotation = mock.MagicMock(userid='acct:luke@hypothes.is')
+        thread_ids = []
+        annotation = mock.MagicMock(userid='acct:luke@hypothes.is',
+                                    thread_ids=thread_ids)
```

Comment:
@sheetaluk So `annotation` here is a mock of a `models.Annotation` class? Or some other class? We should try to use `create_autospec()` to create mocks, and avoid the perils of calling `Mock()` or `MagicMock()` directly, if at all possible. For details see this post: <https://www.seanh.cc/posts/the-problem-with-mocks>

Reply:
Since there are other tests which use `mock.MagicMock` within this module, I'll send a followup PR to correct that usage instead of cluttering this PR.